docs/IndexInvalidation.rst

```diff
@@ -34,7 +34,7 @@ point of program execution.  A given index can be valid for zero, one or more
 than one collection instance at the same time.
 
 An index that is valid for a certain collection designates an element of that
-collection or represents a one-past-end index.
+collection or represents an one-past-end index.
```

Comment:
I'm not an expert in this, but I think it was correct before.

> You use an, not before a word beginning with vowels, but beginning with the sound of a vowel. 

https://www.quora.com/Which-is-correct-a-one-or-an-one